src/test/kotlin/uk/gov/justice/digital/assessments/services/DocumentServiceTest.kt

```diff
@@ -1,121 +0,0 @@
-package uk.gov.justice.digital.assessments.services
-
-import io.mockk.every
```

Comment:
It looks like the content of this file was deleted, but possibly not the file itself? Please can you check if it needs deleting?

Reply:
👍